### PR TITLE
Replaced matching game with version that supports the migration article

### PR DESCRIPTION
**Pull request by adegeo**
- Uploaded the .NET Framework version
- Converted the framework version to .NET 5

### CODE REVIEW

**Comment by adegeo** [comment]
@tdykstra

**Comment by Tanya-Solyanik** [comment]
Was VB sample removed intentionally, or is it coming soon? We also support VB on .NET 5.0, so it would be good to have a vb sample under .NET 5.

In terms of updating the target framework, change look reasonable. Had these samples been buddy-tested?

**Comment by adegeo** [comment]
@Tanya-Solyanik I took the sample from @OliaG tutorial article and repo. VB will be coming next.

**Review by Tanya-Solyanik — APPROVED** [review]

File: windowsforms/matching-game/net45/cs/MatchingGame/MatchingGame.csproj [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{33E7C299-4ABD-4224-9335-3BC70512A1B3}</ProjectGuid>
+    <OutputType>WinExe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>MatchingGame</RootNamespace>
+    <AssemblyName>MatchingGame</AssemblyName>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```
[the remark this answers is not in the document]
Comment (adegeo) [reply windowsforms/matching-game/net45/cs/MatchingGame/MatchingGame.csproj:12]: That's what the original project was using, I didn't change the original project.

**Review by tdykstra — APPROVED** [review]
LGTM, just a few nits noted.

File: windowsforms/matching-game/net45/cs/readme.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- dotnet-framework
+- windows-forms
+page_type: sample
+name: "Matching Game Sample (.NET Framework)"
+urlFragment: "matching-game-net45-csharp"
+description: "A simple matching game "
+---
+# Matching game
+
+This sample demonstrates a simple memory game where you try to match tiles in the fewest amount of moves possible. This sample was used as the basis of a migration tutorial and was migrated to [.NET 5](https://github.com/dotnet/samples/tree/master/windowsforms/matching-game/net5-windows/cs).
```
[the remark this answers is not in the document]
Comment (adegeo) [reply windowsforms/matching-game/net45/cs/readme.md:14]: Good catch

File: windowsforms/matching-game/net45/cs/readme.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Prerequisites
+
+- [Visual Studio 2019 version 16.8 Preview](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019+desktopguide+winforms)
```
[the remark this answers is not in the document]
Comment (adegeo) [reply windowsforms/matching-game/net45/cs/readme.md:18]:
```suggestion
- [Visual Studio 2019 version 16.8](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019+desktopguide+winforms)
```

File: windowsforms/matching-game/net5-windows/cs/readme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+urlFragment: "matching-game-net-csharp"
+description: "A simple matching game "
+---
+# Matching game
+
+This sample demonstrates a simple memory game where you try to match tiles in the fewest amount of moves possible. This sample was migrated the [.NET Framework version](https://github.com/dotnet/samples/tree/master/windowsforms/matching-game/net45/cs).
+
+## Prerequisites
+
+- [Visual Studio 2019 version 16.8 Preview](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019+desktopguide+winforms)
```
[the remark this answers is not in the document]
Comment (adegeo) [reply windowsforms/matching-game/net5-windows/cs/readme.md:19]:
```suggestion
- [Visual Studio 2019 version 16.8](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019+desktopguide+winforms)
```

**Pull request merged**